airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/jsonl/S3JsonlFormatConfig.java

```diff
@@ -4,30 +4,39 @@
 
 package io.airbyte.integrations.destination.s3.jsonl;
 
-import static io.airbyte.integrations.destination.s3.S3DestinationConstants.COMPRESSION_ARG_NAME;
-import static io.airbyte.integrations.destination.s3.S3DestinationConstants.DEFAULT_COMPRESSION_TYPE;
-import static io.airbyte.integrations.destination.s3.S3DestinationConstants.FLATTEN_DATA;
+import static io.airbyte.integrations.destination.s3.S3DestinationConstants.*;
```

Comment:
Minor nit: In the IntelliJ code style there is a section that prevents the code editor (IntelliJ) from auto-importing with `*`

Context: Under [Section 5](https://docs.airbyte.com/contributing-to-airbyte/code-style#configure-java-style-for-intellij)

The reason for bringing this up is to keep consistency across import statements but it also makes it clear to the user what functions/fields is being imported